tests/page-objects/forms/z-score.po.js

```diff
@@ -133,65 +134,64 @@ const docs = [
     charts: charts
   }];
 
-const clearAndFill = (el, value) => {
-  return el.clear().then(() => el.sendKeys(value));
+const clearAndFill = async (el, value) => {
+  await el.clear();
+  await el.sendKeys(value);
 };
 
-const clickAndGetValue = el => {
-  el.click();
-  return el.getAttribute('value');
+const clickAndGetValue = async el => {
+  await el.click();
+  return await el.getAttribute('value');
 };
 
 module.exports = {
-  configureForm: (userContactDoc, done) => {
-    utils.seedTestData(done, userContactDoc, docs);
+  configureForm: async (userContactDoc) => {
+    await utils.seedTestDataNative(userContactDoc, docs);
   },
 
-  load: () => {
-    common.goToReports();
-    browser.wait(() => {
-      return element(by.css('.action-container .general-actions:not(.ng-hide) .fa-plus')).isPresent();
-    }, 10000);
+  load: async () => {
+    await common.goToReportsNative();
+    await helper.waitUntilReadyNative(element(by.css('.action-container .general-actions:not(.ng-hide) .fa-plus')));
```

Comment:
Followup from conversation @newtewt and I had via Meet (in the the case others are reading along on this one):

I put my comment on the wrong line here, causing quite a bit of confusion. What @newtewt has here using the `await` **is correct**. The last statement in this function is the one that should just return the Promise (rather than `await`ing it).

Marking this conversation as resolved as @newtewt knows what he's doing and we're all good here. Sorry for creating the confusion!